framework/include/constraints/MortarData.h

```diff
@@ -12,14 +12,17 @@
 #include "AutomaticMortarGeneration.h"
 #include "MooseHashing.h"
 
-#include <map>
+#include "libmesh/parallel_object.h"
+
+#include <unordered_map>
+#include <set>
 
 class SubProblem;
 
-class MortarData
+class MortarData : public libMesh::ParallelObject
```

Comment:
Just curious why this does not derive from MooseObject?

Reply:
I just didn't see the need yet to go through and create `InputParameters` and all that jazz. If this starts taking more construction arguments, then I can see changing its inheritance to `MooseObject`